src/content/en/updates/2020/09/migrating-to-es-modules.md

```diff
@@ -0,0 +1,256 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: How we migrate Chrome DevTools to ES modules. The Chrome DevTools engineering blog - by the developers who build the DevTools.
+
+{# wf_updated_on: 2020-09-08 #}
+{# wf_published_on: 2020-09-08 #}
+{# wf_tags: devtools-blog #}
+{# wf_featured_image: /web/updates/images/generic/chrome-devtools.png #}
+{# wf_featured_snippet: How we migrate Chrome DevTools to ES modules. #}
+{# wf_blink_components: N/A #}
+
+# Migrating to ES modules {: .page-title }
```

Comment:
Since this is a series, do we want to name the title to include a series name. e.g. "DevTools architecture refresh: Migrating to ES modules"? @TimvdLippe

Reply:
Sounds like a good idea!